com.unity.render-pipelines.universal/Documentation~/ShaderQuality.md

```diff
@@ -0,0 +1,19 @@
+# Shader Quality
+The shader quality can be set to be one of three settings __Low__, __Medium__ or __High__.
+## Default values:
+If the shader quality is not defined at compile time then the shader quality will be defined to be:
+__Low__ if GLES is targeted. __Medium__ if moblie platfrom is targeted. __High__ is chosen if the other 2 does not apply to the target.
```

Comment:
"platform" is misspelled here, as is "mobile"

Reply:
I think the term "GLES" is not clear. I this context is means "OpenGL ES 2.0/WebGL 1.0".

Reply:
This should be fixed now